rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -1125,6 +1125,9 @@ stream {
             {{- range $line := buildAuthResponseHeaders $externalAuth.ResponseHeaders }}
             {{ $line }}
             {{- end }}
+            {{- range $line := buildAuthProxySetHeaders $externalAuth.ProxySetHeaders}}
+            {{ $line }}
+            {{- end }}
```

Comment:
Should this not be in https://github.com/kubernetes/ingress-nginx/pull/4550/files#diff-980db9e4b88704f12338bd074839f94eR868?

Reply:
You're right, this was sending the headers to the backend :/